egs/wham/ConvTasNet/run.sh

```diff
@@ -50,9 +37,22 @@ mask_nonlinear=relu
 # Evaluation
 eval_use_gpu=1
 
+# Main storage directory. You'll need disk space to dump the WHAM mixtures and the wsj0 wav
+# files if you start from sphere files.
+storage_dir=corpus
 
 . utils/parse_options.sh
 
+# If you start from the sphere files, specify the path to the directory and start from stage 0
+sphere_dir=$storage_dir/sph_files
+# If you already have wsj0 wav files, specify the path to the directory here and start from stage 1
+wsj0_wav_dir=$storage_dir/wsj0_wavs
+# If you already have the WHAM mixtures, specify the path to the directory here and start from stage 2
+wham_wav_dir=$storage_dir/wham_wavs
```

Comment:
Can you elaborate on what was the problem with the previous setting? 
IMO, having the storage options at the top makes more sense.

Reply:
And what if somebody starts from stage 1 with wsj0 wavs, but the folder name isn't called wsj0_wavs? Then the user has to scroll until here and change it by hand?